ostree-upload-server.py

```diff
@@ -128,9 +139,33 @@ def upload(self):
                 os.close(f)
                 upload.save(real_name)
 
-                task = ReceiveTask(upload.filename, real_name,
-                                   self._repo)
+                repo_name = request.form.get('target-repo', None)
```

Comment:
Why is this in the form? Wouldn't it be better as a parameter? Also, I think `repo` is all that's needed since I can't see you specifying anything besides the repository to upload to.